docs/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+title: "BUG: "
+labels: bug-report "P3 - Normal"
+assignees: kreyren nntoan
+---
+
+### Actual Behaviour
+<!-- Describe how the bug manifests. -->
+
+
+
+### Expected Behaviour
+<!-- State here what the feature should enable the user to do. -->
+
+
+
+### Steps to reproduce it
+<!-- Add steps to reproduce provided bug, BE VERBOSE! -->
+
+
+
+### System information:
+<!-- Add information about the system your facing this bug on. If you think this is irrelevant or if it's a UI bug or a feature request, please remove this section -->
```

Comment:
Is there a `omb` function we could use to assist users in getting this information? `asdf` has a `asdf info` function which outputs...
```
OS:
Linux Earth 5.15.0-119-generic #129-Ubuntu SMP Fri Aug 2 19:25:20 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

SHELL:
GNU bash, version 5.1.16(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

BASH VERSION:
5.1.16(1)-release

ASDF VERSION:
v0.14.1
```

Reply:
We recently added a skeleton `omb`, but no subcommands are implemented. We may check the upstream OMZ to see if they have an implementation of `omz info` or similar.

Reply:
The closest in the upstream seems to be [`omz version`](https://github.com/ohmyzsh/ohmyzsh/blob/master/lib/cli.zsh#L819), but this only shows the version of Oh My Zsh.